index.html

```diff
@@ -1391,6 +1402,16 @@
           used to probe information about which <a>remote playback device</a>
           the user has without user consent.
         </p>
+        <p>
+          If a user <a data-lt="disable remote playback">disables remote
+          playback</a> on all <a data-lt="media element">media elements</a>
+          (through a browser setting, for example), the user agent should
+          stop <a data-lt="monitor the list of available remote playback devices">
+          monitoring the list of available remote playback devices</a>
+          and will clear the <a>global set of availability callbacks</a>. In
+          this state, pages will receive no further information about the
```

Comment:
This seems broken if the last known information was that the availability is true. Also, there's no way for the pages to know the callback ids became invalid. I think in this case the UA should just report the availability as false and stop reporting availability through the callbacks.